lua/telescope/actions/init.lua

```diff
@@ -805,6 +805,213 @@ actions.cycle_previewers_prev = function(prompt_bufnr)
   actions.get_current_picker(prompt_bufnr):cycle_previewers(-1)
 end
 
+--- Display the keymaps of registered actions similar to which-key.nvim.<br>
+--- - Notes:
+---   - Convenience function to provide configuration options at setup and not meant to be called directly.
+---   - See |actions._toggle_registered_actions| for full documentation.
+---@param opts table: options to pass to toggling registered actions
+actions.toggle_registered_actions = function(opts)
```

Comment:
what is the purpose of this function?

I think it should be called something like `create_...` or some other name that shows you are returning a function, not doing the action.